agent/src/agent/pipeline/config/expression_parser/condition.py

```diff
@@ -70,6 +108,35 @@ def validate(condition: str) -> bool:
     return True
 
 
+def validate_value(value: str):
```

Comment:
this is not used anywhere

Reply:
what do you mean?
https://github.com/anodot/daria/blob/6d95e42dc46b36219a4d5dba4dcfa0a1b57d36fc/agent/src/agent/pipeline/config/expression_parser/transformation.py#L11